src/document/types.ml

```diff
@@ -61,11 +61,13 @@ and Block : sig
 
   and one = { attr : Class.t; desc : desc }
 
+  and description_item = { item_attr : Class.t; term : Inline.t; def : t }
+
   and desc =
     | Inline of Inline.t
     | Paragraph of Inline.t
     | List of list_type * t list
-    | Description of (Inline.t * t) list
+    | Description of description_item list
```

Comment:
I would suggest adding a `description` module, with types `one` and `t`, and then you can use `attr` as label again. Also, I would prefer,`key` and `definition` for the other field names.